github.js

```diff
@@ -13,11 +13,10 @@ var zlib = require('zlib');
 
 var semver = require('semver');
 
-var which = require('which');
-
 function extend(dest, src) {
   for (var key in src) {
-    dest[key] = src[key]
+    if (key in dest && typeof dest[key] === 'object') extend(dest[key], src[key]);
```

Comment:
Just `typeof dest[key] === 'object'` should be ok here as that implies `key in dest`?